reference/enterprise-api-ref/federated-reporting-api.markdown

```diff
@@ -0,0 +1,226 @@
+---
+layout: default
+title: Federated reporting configuration API
+published: true
+tags: [reference, enterprise, API, reporting]
+---
+
+Federated reporting API allows to configure federated reporting process.
+
+# Remote hubs
+
+## Remote hubs list
+
+**URI:** https://hub.cfengine.com/api/fr/remote-hub
+
+**Method:** GET
+
+**Example response:**
+
+```
+HTTP 200 OK
+{
+    "id-1": {
+        "id": 1,
+        "hostkey": "SHA=2d67a6840878de...",
+        "api_url": "https://172.28.128.5",
+        "ui_name": "ubuntu-xenial",
+        "role": "feeder",
+        "target_state": "on",
+        "transport": {
+            "mode": "push_over_rsync",
+            "ssh_user": "cfdrop",
+            "ssh_host": "172.28.128.5",
+            "ssh_pubkey": "",
+            "ssh_fingerprint": ""
+        }
+    },
+    "id-2": {
+        "id": 2,
+        "hostkey": "SHA=wefweg34tgfds...",
+        "api_url": "https://172.28.128.6",
+        "ui_name": "production s",
```

Comment:
this ui_name is a bit odd, "production s" ?

Reply:
renamed